learn-to-code-variable-fonts.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Getting started with variable fonts"
+date: 2021-21-03
+author: "federicab"
+category: "learn to code"
+tags:
+    - post
+    - guest-post
+layout: article
+preview: "The next big thing in web typography"
+featured: false
+
+---
+
+You might not be a web typography freak, but no developer likes a FOUT (Flash Of Unstyled Text) welcoming users on their pages. Variable fonts are here to help. And if you really are the web typography freak type you will be even more enticed to hear about variable fonts. So let’s get started!
```

Comment:
maybe change the "the" into an "a" in the following line: 
if you really are the web typography freak 
and drop the "type" after that sentence.